src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -592,7 +592,7 @@ ProcessCLRException(IN     PEXCEPTION_RECORD   pExceptionRecord,
                 // The 3rd argument passes to PopExplicitFrame is normally the parent SP to correctly handle InlinedCallFrame embbeded
                 // in parent managed frame. But at this point there are no further managed frames are on the stack, so we can pass NULL.
                 // Also don't pop the GC frames, their destructor will pop them as the exception propagates.
-                // NOTE: this needs to be popped in the 2nd pass to ensure that crash dumps and Watson get the dump with these still 
+                // NOTE: this needs to be popped in the 2nd pass to ensure that crash dumps and Watson get the dump with these still
```

Comment:
My editor stripped trailing whitespace from the file. Feel free to just disable ws-only changes in this diff.